frontend/lib/pages/issue-pages.tsx

```diff
@@ -123,14 +125,16 @@ function IssueAreaLink(props: IssueAreaLinkProps): JSX.Element {
       {(ctx) => {
         const count = areaIssueCount(area, ctx.session.issues as IssueChoice[], ctx.session.customIssuesV2 || []);
         const url = props.routes.area.create(allCapsToSlug(area));
+        const modalUrl = props.routes.modal;
+        const hadViewedModal = browserStorage.get('hasViewedCovidRiskModal');
```

Comment:
Hmm, I don't think we should be pulling from browser storage at render time, because that's going to result in an initial render inconsistency between server and client (this `hadViewedModal` is always going to return `undefined` on the server-side, while it might return `true` on the client on its first render).  Hmm.  If this is a functional component, we could add a react hook called `useBrowserStorage` that always returns `undefined` on the first render but then immediately returns an updated value from browser storage when mounted on the client side.

Also, I'm not sure if this is related, but when reloading `/loc/issues` in my browser I'm seeing the following error logged:

```
Warning: Prop `href` did not match. Server: "/loc/issues/covid-risk-modal" Client: "/loc/issues/home"
```

That's particularly weird because neither of those URLs is the URL I'm actually at!

Reply:
Err another funky thing is that when I click "I understand the risk" the modal disappears, but I'm not taken into the apartment area I had originally clicked on, so I have to click on it again to actually go into the area.  That feels weird, is it intentional?

Reply:
Fixed!

Reply:
NOICE thank you!